src/main/java/deku/UI.java

```diff
@@ -0,0 +1,11 @@
+package deku;
+
+class UI {

```

Comment:
UI should be named as Ui instead.
> Abbreviations and acronyms should not be uppercase when used as a part of a name.
```suggestion
class Ui {
```